etc/config-private.pl

```diff
@@ -20,7 +20,7 @@
                'master' => {  # master must be named 'master'
                    'host' => "localhost",
                    'port' => 3306,
-                   'user' => 'dw',
+                   'user' => 'dw',    #CHANGETHIS if on Dreamhack
                    'pass' => 'password',    # CHANGETHIS
                    'dbname' => 'dw',
```

Comment:
Might as well comment dbname as well, if you want to make a note to change all the things that need to be changed. Pretty sure all the database names on the Dreamhack server are of the form "dreamhack_[username]".